src/ai/defaultai.cc

```diff
@@ -354,11 +355,15 @@ void DefaultAI::think() {
 	std::sort(current_task_queue.begin(), current_task_queue.end());
 
 	// Performing tasks from temporary queue one by one
-	for (const SchedulerTask& task : current_task_queue) {
+	for (auto task : current_task_queue) {
```

Comment:
clang-tidy is complaining to make this const

Reply:
Should be fixed. This is beyond my C++ capabilities though :(

Reply:
> Should be fixed. This is beyond my C++ capabilities though :(

I don't understand this completely as well however I normally just do what clang-tidy suggests.

Reply:
fixed